lib/engine/game/g_18_carolinas.rb

```diff
@@ -7,6 +7,57 @@ module Engine
   module Game
     class G18Carolinas < Base
       load_from_json(Config::Game::G18Carolinas::JSON)
+
+      def init_companies(players)
+        super.take(players.size).each do |company|
```

Comment:
why are you doing each do? remove this if unnedded

Reply:
okay, yeah, I think it isn't doing anything